### ISSUE TITLE
jenkins: mac test failed

### ISSUE THREAD

**Issue by pinggao187** [intel/analytics-zoo#336]
issue: [#3719](https://github.com/intel-analytics/analytics-zoo-internal/issues/339)

I tried to run all packages on mac, and many test cases failed.
jenkins link:
http://10.239.47.210:18888/view/ZOO-NB/job/ZOO-NB-UnitTests-2.4-SCALA-MAC/361/

All Failed Tests
com.intel.analytics.zoo.pipeline.api.NetSpec.Save to tensorflow should works | 2.7 sec | 1
com.intel.analytics.zoo.pipeline.api.NetSpec.graph definition should work | 2.3 sec | 1
com.intel.analytics.zoo.pipeline.api.keras.layers.BertSpec.Bert should be able to work | 0.22 sec | 1
com.intel.analytics.zoo.pipeline.api.keras.layers.BertSpec.Bert with output all blocks should be able to work | 8 ms | 1
com.intel.analytics.zoo.pipeline.api.keras.layers.BertSpec.Bert with configed embedding should be able to work | 8 ms | 1
com.intel.analytics.zoo.pipeline.api.keras.layers.BertSpec.BERT should be able to generate correct result | 7 ms | 1
com.intel.analytics.zoo.pipeline.api.keras.layers.BertSpec.Bert should save/load be able to work | 66 ms | 1
com.intel.analytics.zoo.pipeline.api.keras.layers.BertSpec.Bert gelu should be able to generate correct result | 4 ms | 1
com.intel.analytics.zoo.pipeline.api.keras.layers.Deconvolution2DSpec.Deconvolution2D without bias should be the same as Keras | 4.5 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.AveragePooling1DSpec.AveragePooling1D same mode should be the same as Keras | 4.1 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.Conv1DSpec.Conv1D without bias should be the same as Keras | 4 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.Conv2DSpec.Conv2D NCHW should be the same as Keras | 4.5 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.Conv2DSpec.Conv2D NHWC should be the same as Keras | 4.3 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.Conv2DSpec.Conv2D without bias should be the same as Keras | 4.5 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.DenseSpec.Dense nD input should be the same as Keras | 4 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.GlobalAveragePooling2DSpec.GlobalAveragePooling2D NCHW should be the same as Keras | 4.5 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.GlobalAveragePooling2DSpec.GlobalAveragePooling2D NHWC should be the same as Keras | 4.4 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.GlobalAveragePooling3DSpec.GlobalAveragePooling3D should be the same as Keras | 4.5 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.GlobalMaxPooling2DSpec.GlobalMaxPooling2D NCHW should be the same as Keras | 4 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.GlobalMaxPooling2DSpec.GlobalMaxPooling2D NHWC should be the same as Keras | 4.1 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.GlobalMaxPooling3DSpec.GlobalMaxPooling3D should be the same as Keras | 4 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.MaxPooling1DSpec.MaxPooling1D valid mode should be the same as Keras | 4 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.MaxPooling1DSpec.MaxPooling1D same mode should be the same as Keras | 4.1 sec | 1
com.intel.analytics.zoo.pipeline.api.keras2.layers.Softmax.softmax with 4d input should be the same as Keras | 4.2 sec | 1
